config.md

```diff
@@ -244,7 +244,7 @@ This specification uses the following terms:
             Implementations MUST generate an error if they encounter a unknown value while verifying or unpacking an image.
           </li>
           <li>
-            <code>diff_ids</code> is an array of layer content hashes (<code>DiffIDs</code>), in order from bottom-most to top-most.
+            <code>diff_ids</code> is an array of layer content hashes (<code>DiffIDs</code>), in order from bottom-most to top-most and bottom-most at index 0.
```

Comment:
#318 is proposing consolidated language for the manifest's `layers`.  Having `layers` in one JSON blob and `diff_ids` in another is awkward, but I'd suggest leaning on the `layers` definition with something like:

    <code>diff_ids</code> has the same semantics as the <a href="manifest.md#image-manifest-property-descriptions">manifest's `layers`</a>, except that the digests MUST all be taken after removing any compression.

Without `layers`, `diff_ids` seems pretty useless.  How do you recover the type of the referenced blob?  Can we land the `+gzip` suffix being discussed in #316 and convert `diff_ids` to an array of descriptors?  Then you could tell that one entry pointed at an`application/vnd.oci.image.layer.tar` while another pointed at an `application/vnd.oci.image.layer.nondistributable.tar` and another pointed at an `application/vnd.example.image.extension`.  Then `diff_ids` would have a shot at being useful in its own right (in cases where the consumer had access to a CAS engine holding the uncompressed blobs).